grails-app/controllers/au/org/emii/aatams/SensorController.groovy

```diff
@@ -32,8 +32,6 @@ class SensorController extends ReportController {
         def tag = tagFactoryService.lookupOrCreate(params.tag)
         assert(tag)
 
-        params.tag = tag
```

Comment:
Problem occurs when using the Create Tag (or Sensor) page to add a new sensor to a tag that already exists, changing the device model at the same time

This fix fixes that problem.  We should rely on existing tests to verify other problems.   Might be worth adding this particular scenario to a test to ensure that we don't revert this fix in future.